docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,983 @@
+# Bold BI auto deployment using kubectl on Kubernetes Service
+
+This section helps you to deploy [Bold BI](https://www.boldbi.com/) in Kubernetes without manually activating licensing and configuring startup from the browser. This also helps us to customize the branding using environment variables.
+
+## Deployment Methods
+
+There are two ways to deploy Bold BI on the Kubernetes cluster. Please refer to the following documents for Bold BI deployment.
```

Comment:
There are two ways to deploy Bold BI on the Kubernetes cluster. Please refer to the following documents for Bold BI deployment:

Reply:
concern addressed